ios/FluentUI/Table View/TableViewCell.swift

```diff
@@ -146,9 +139,15 @@ open class TableViewCell: UITableViewCell, TokenizedControlInternal {
      `mediumHeight` - Height for the cell when only the `title` and `subtitle` are provided in 2 lines of text.
      `largeHeight` - Height for the cell when the `title`, `subtitle`, and `footer` are provided in 3 lines of text.
      */
-    @objc public static var smallHeight: CGFloat { return height(title: "", customViewSize: .small) }
-    @objc public static var mediumHeight: CGFloat { return height(title: "", subtitle: " ") }
-    @objc public static var largeHeight: CGFloat { return height(title: "", subtitle: " ", footer: " ") }
+    @objc public static var smallHeight: CGFloat {
+        return 48
+    }
+    @objc public static var mediumHeight: CGFloat {
+        return 64
+    }
+    @objc public static var largeHeight: CGFloat {
```

Comment:
Why do we need to define these values here if they are also defined in `TableViewCellTokens`?

Reply:
Thanks for bringing this up, I meant to open a thread about this! These are static variables, which means they can be accessed without actually instantiating the TableViewCell. I'm guessing that this is being used by downstream dependencies, and since they're variables, I can't change them to methods. They could also use the tokens to obtain these values from here on out. For now I put in the actual CGFloat value, but maybe I can add deprecation messages/comments like I did for the custom text color methods? @harrieshin @rdeassis

Reply:
Resolved offline: removed labels which allowed me to revert to original code